doc/bluealsa-api.txt

```diff
@@ -21,6 +21,176 @@ Signals         void PCMAdded(object path, dict props)
                         object path is only for a reference - it has been
                         already removed.
 
+Properties      string Version [readonly]
+
+                        Release version of BlueALSA service.
+
+                array{string} Profiles [readonly]
+
+                        Registered Bluetooth Profiles.
+
+                        Possible values: "A2DP-sink", "A2DP-source",
+                                         "HFP-ofono", "HFP-AG", "HFP-HF",
+                                         "HSP-AG" or "HSP-HS"
+
+                array{string} Adapters [readonly]
```

Comment:
As a first commit, which will introduce properties, I will keep `Version` and `Adapters`. `Version` is a nice to have property. Where, used `Adapters` is something which is not known unless BlueALSA service is running.

`AdapterFilter` is defined via command line, so the purpose of exposing it via D-Bus API would be only as a debug information. But, IMHO, it is better not to clutter API with debug-only properties - stderr/syslog is for debug output.

As for other properties I will have to think a little bit how to structure them. Because currently there is some inconsistency: HFP has sub-properties, where A2DP has not. Also, I'm not sure whether they should be there at all...

`FeaturesSDPHF`, `FeaturesSDPAG`, etc. is more like a configuration. However, I like the verbose output of `FeaturesRFCOMM`, so maybe it will be a good idea to extend RFCOMM properties.